primitives/Cargo.toml

```diff
@@ -24,7 +24,7 @@ std = [
     "orml-tokens/std",
     "orml-traits/std",
     "parity-scale-codec/std",
-    "serde",
+    "serde/default",
```

Comment:
Why not directly `std`?

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1059/commits/d646156a5e1689a59c0399b7aae9398ceca92668